module/documents/item.mjs

```diff
@@ -1206,16 +1204,16 @@ export default class Item5e extends SystemDocumentMixin(Item) {
    * @returns {ItemUseConfiguration}  Configuration data for the roll.
    */
   _getUsageConfig() {
-    const { consume, uses, enchantment, summons, target, level, preparation } = this.system;
+    const { consume, uses, summons, target, level, preparation } = this.system;
 
     const config = {
-      applyEnchantment: null,
       createMeasuredTemplate: null,
       createSummons: null,
       consumeResource: null,
       consumeSpellSlot: null,
       consumeUsage: null,
       enchantmentProfile: null,
+      promptEnchantment: null,
```

Comment:
We should probably add this to the `ItemUseConfiguration` typedef if it's a property that can be returned from this method.